cou/utils/openstack.py

```diff
@@ -240,20 +240,20 @@ def codename(self, value: str) -> None:
             raise ValueError(f"OpenStack '{value}' is not in '{self.openstack_codenames}'")
 
     @property
-    def next_release(self) -> Optional[str]:
+    def next_release(self) -> Optional["OpenStackRelease"]:
```

Comment:
It seems that in your next patch you didn't update the return type. I hope the lint is able to get it.

Reply:
Hmm no, the lint didn't catch it when I was running it locally, not even when I was trying to change the return type in docstring to something random. Strange. But I just updated the docstring.

Reply:
What if benefit to use `from __future__ import annotations` instead of `"OpenStackRelease"`? I always using it with quotes, e.g. ``"OpenStackRelease"``

Reply:
using `OpenStackRelease` lint was able to find that the docstring was not updated.